### PR TITLE
Update the DuckDB version to 1.1.0

### PR DESCRIPTION
**Pull request by mrigger**
Most of the individual commits contain detailed information about the changes.

### CODE REVIEW

**Comment by mrigger** [comment]
@malwaregarry, could you please help review this PR?

File: src/sqlancer/duckdb/DuckDBErrors.java [hunk 1]
```diff
@@ -63,20 +60,17 @@ public static List<String> getExpressionErrors() {
...
-        errors.add("missing ]");
-        errors.add("missing )");
-        errors.add("invalid escape sequence");
-        errors.add("no argument for repetition operator: ");
-        errors.add("bad repetition operator");
-        errors.add("trailing \\");
-        errors.add("invalid perl operator");
-        errors.add("invalid character class range");
-        errors.add("width is not integer");
+        errors.add("Invalid Input Error:");
```
Comment (malwaregarry) [inline src/sqlancer/duckdb/DuckDBErrors.java:72]: there's a duplicated line here

**Comment by malwaregarry** [comment]
I noticed that the CI is failing. Do you need help investigating?

**Review by malwaregarry — APPROVED** [review]
LGTM other than the small nit

File: src/sqlancer/duckdb/DuckDBErrors.java (hunk 1 above)
Comment (mrigger) [reply src/sqlancer/duckdb/DuckDBErrors.java:72]: Thanks, fixed this.

**Comment by mrigger** [comment]
> I noticed that the CI is failing. Do you need help investigating?

If you have time to look into this, it would be great. I think this is likely a logic bug in DuckDB - not sure if it still triggers for the latest version.

**Comment by malwaregarry** [comment]
@mrigger I updated to v1.1.0 and added a new bug https://github.com/duckdb/duckdb/issues/13933

**Comment by mrigger** [comment]
Great, thanks!

**Pull request merged**